pages/releases/4.3.html

```diff
@@ -0,0 +1,2076 @@
+								<div class="c-blockquote">
+									A useful <span class="highlight">reminder</span>
+								</div>
+								<div class="c-content">
+									<p>
+										A new warning has been added when .cs files have been modified after the last
+										build.
+									</p>
+									<p>
+										You'll now never miss when a re-build is needed.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="paulloz"
+									data-read-more="https://github.com/godotengine/godot/pull/85869">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<img class="release-card-image position-top-left"
+								src="/storage/releases/4.3/images/dotnet-rebuild-notification.webp"
+								alt="Image of the new re-build notification." draggable="false">
+						</div>
+					</div>
+				</div> <!-- #dotnet-rebuild-notification -->
+
+				<div id="dotnet-more-robust-reload-process" class="release-card span-2">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#dotnet-more-robust-reload-process">More robust reload process</a>
+								</h4>
+								<div class="c-blockquote">
+									<span class="highlight">More robust</span> than ever
+								</div>
+								<div class="c-content">
+									<p>
+										One of the most annoying issues that C# users have with Godot have been fixed.
+									</p>
+									<p>
+										When a C# project is re-built, scripts need to be reloaded. Unfortunately, a
+										lot of things can go wrong during this process. It usually left users in an
+										invalid state with no other option but to re-open the Godot editor entirely.
+									</p>
+									<p>
+										Now, the process has been improved and users will be able to appreciate the new
+										robustness of the reloads.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="paulloz"
+									data-read-more="https://github.com/godotengine/godot/pull/83217">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #dotnet-more-robust-reload-process -->
+
+				<div id="dotnet-better-support-generic-csharp-classes" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#dotnet-better-support-generic-csharp-classes">
+										<span>Better support of generic C# classes</span>
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										This feature has been requested for a long time, and it's finally here.
+									</p>
+									<p>
+										While the implementation has some limitations, you'll be finally able to use
+										(more comfortably) generic C# classes in your code.
+									</p>
+								</div>
```

Comment:
@raulsntos @paulloz Is this alright?

Reply:
I'm a bit wary of the title "Generics now supported out of the box", since Godot doesn't support exposing generic types or methods (and never will).

And to be honest, you could always use generics in your own code (as long as it doesn't reach Godot). The problem that was fixed is using generic classes as the base type[^1] of a non-generic class that is exposed to Godot.

This means you can now have generic classes in the type hierarchy, but generic classes still can't be attached as scripts (it wouldn't make sense, since we wouldn't know what types to use for the generic type parameters).

I'm sorry I'm struggling to come up with a good title that summarizes this. Maybe something that mentions the editor being more robust/stable with type hierarchies that include generic types?

[^1]: Or to put it another way: inheriting from generic classes.